pkg/cluster/internal/delete/delete.go

```diff
@@ -33,16 +33,17 @@ func Cluster(logger log.Logger, p providers.Provider, name, explicitKubeconfigPa
 		return errors.Wrap(err, "error listing nodes")
 	}
 
+	logger.V(0).Infof("Removing context %q from kubeconfig ...", name)
 	kerr := kubeconfig.Remove(name, explicitKubeconfigPath)
 	if kerr != nil {
 		logger.Errorf("failed to update kubeconfig: %v", kerr)
+		return err
 	}
 
+	logger.V(0).Infof("Deleting nodes for cluster %q ...", name)
 	err = p.DeleteNodes(n)
 	if err != nil {
-		return err
-	}
-	if kerr != nil {
+		logger.Errorf("failed to delete nodes: %v", kerr)
```

Comment:
This is redundant to returning the error.

Reply:
Removed.

Reply:
still redundant.